### PR TITLE
Allow version managed app css

### PR DESCRIPTION
**Pull request by stevenadams**
Ticket: NDPE-37

* Add a file for custom CSS to live in
* Make custom CSS available by default to all pages in main layout file
* Modify gulp commands to watch files by default
* Add documentation around adding custom CSS

### CODE REVIEW

File: docs/guides/adding-custom-css.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#Adding custom CSS
```
Comment (ghost) [inline docs/guides/adding-custom-css.md]: Add header for consistency with other pages

File: app/views/layouts/nhsuk_layout.hbs [hunk 2]
```diff
@@ -10,6 +10,7 @@
   <!-- <link rel="apple-touch-icon" href="icon.png"> -->
 
   <link rel="stylesheet" href="/nhsuk.css">
+  <link rel="stylesheet" href="/app.css">
```
Comment (visualunderground) [inline app/views/layouts/nhsuk_layout.hbs:13]: Concat files?

File: server.js [hunk 3]
```diff
@@ -126,7 +126,7 @@ utils.findAvailablePort(app, function (port) {
         ui: false,
         files: ['public/**/*.*', 'app/views/**/*.*'],
         ghostmode: false,
-        open: true,
+        open: false,
```
Comment (visualunderground) [inline server.js:129]: Thank you!

File: readme.md [hunk 4]
```diff
@@ -16,7 +16,8 @@ You must protect user privacy at all times, even when using prototypes. Always m
 2. [Creating Your First Prototype](/docs/guides/your-first-prototype.md)
 3. [Server-side Logic](/docs/guides/server-side-logic.md)
 4. [Page Transitions](/docs/guides/page-transitions.md)
-5. [Providing Data Using Asynchronous APIs](/docs/guides/asynchronous-apis.md)
-6. [Accessibility testing](/docs/guides/accessibility-testing.md)
-7. [Publishing to Heroku](/docs/guides/publishing-to-heroku.md)
-8. [Debugging](/docs/guides/debugging.md)
+5. [Adding custom CSS](/docs/guides/adding-custom-css.md)
```
Comment (ghost) [inline readme.md:19]: Also add this line to docs/guides/index.md

File: docs/guides/adding-custom-css.md (hunk 1 above)
Comment (ghost) [reply docs/guides/adding-custom-css.md]: Also this isn't rendering as a heading for me

**Review by visualunderground — CHANGES_REQUESTED** [review]
Feature works for me, I’ve tested:

 - Watch command
 - Ensuring styles replicate across both new and existing pages
 - Adding partials

QUESTION:
 - Should we concat the CSS files?

ISSUES:
 - There's an issue with the styling of the header on the docs
 - The file docs/guides/index.md doesn't reference the new CSS doc

File: app/views/layouts/nhsuk_layout.hbs (hunk 2 above)
Comment (stevenadams) [reply app/views/layouts/nhsuk_layout.hbs:13]: I'm not fully decided on the method for merging everything together right now, concat may come in in future after investigation

**Review by visualunderground — APPROVED** [review]

**Pull request merged**